### ISSUE TITLE
Placeholder for ideas for improvement

### ISSUE THREAD

**Issue by possibly-sam** [possibly-sam/mcfc01#1]
* Better filtering
* Static calculation of confidence interval based on assumption of discrete distribution.
* UI improvements
* Integration with other tools, e.g. MS Power BI

**Comment by raminty** [comment]
Static calculations are possible if we assume independent risks, and could be interesting to look at.
Also:
- Highlight to the user (in UI) possible typo mistakes and other outliers in the risk register, along the lines we found while cleaning data
- Experiment with ways of getting users to identify if risks are correlated (dependent), especially useful for making the most of MC simulation

**Comment by possibly-sam** [comment]
Good point.  I am thinking that for #1 it is almost a separate application.  That is, the application has two almost independent parts.  1) Input and clean the data, and 2) Perform calculations and report on clean data.  From that point of view, the typo question might fit in #1.

For #2, i was thinking that if we had detailed descriptions of the risks, we could do a textual analysis for similar terms and assume some correlation.  If two risks both have a phrase like 'LTL Trucking Delivery' in them, it might be reasonable to assume a correlation.

**Comment by raminty** [comment]
Yes, both of those make sense to me. Those textual indicators can also be used to indicate to users if they are possibly duplicating risks that already exist - human-in-the-loop augmentation rather than full automation.

**Comment by possibly-sam** [comment]
*Meeting Room Booked*

Holborn - Meeting room 410
St.Giles, 154 Southampton Row, Holborn, London, WC1B 5AL
Date: Saturday 11 May
Start time: 11:00AM
Duration: 4 Hours
Guests: 4

**Comment by possibly-sam** [comment]
Faster(?) version updated to https://north-loop-designs.shinyapps.io/mcfc03/

**Comment by possibly-sam** [comment]
Initial thoughts on the Black Swan Lever..

[bsl-01.pdf](https://github.com/possibly-sam/mcfc01/files/3244890/bsl-01.pdf)

**Comment by aangt321** [comment]
Guys where is the link to the clean set of data for download & analysis?

**Comment by possibly-sam** [comment]
https://github.com/possibly-sam/mcfc01/blob/master/risk_KA_11.csv

risk_KA_11.csv was the file that Ken prepared and that we used in the last hackathon.  It is the one I have continued to use for all my testing.

[risk_KA_11.zip](https://github.com/possibly-sam/mcfc01/files/3269743/risk_KA_11.zip)
